examples/golang/bucket-notification-deletion/notification-deletion/notification.go

```diff
@@ -0,0 +1,73 @@
+package notification
+
+import (
+	"net/http"
+
+	"github.com/aws/aws-sdk-go/aws/awsutil"
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+type DeleteBucketNotificationRequestInput struct {
```

Comment:
please add an optional "notificationID" parameter

Reply:
How Do I go about adding the NotificationID here?

Reply:
add it to this type. similarly to "Bucket":
```golang
// The name of the notification to delete from the bucket.
//  
// NotificationID is an optional field. If not provided, all notifications are deleted from the bucket
NotificationID *string `location:"uri" locationName:"NotificationID" type:"string" required:"false"`
```